recipes/libgsasl/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.8.0":
```

Comment:
Do you need both versions? Otherwise, keep only the latest. We try to avoid extra versions which are not directly requested by users.